viam-orb-slam3.go

```diff
@@ -57,11 +59,18 @@ const (
 	cameraValidationIntervalSec = 1.
 	parsePortMaxTimeoutSec      = 60
 	// time format for the slam service.
-	slamTimeFormat        = "2006-01-02T15:04:05.0000Z"
 	opTimeoutErrorMessage = "bad scan: OpTimeout"
 	localhost0            = "localhost:0"
 )
 
+// This gets created from the subAlgo parameter
```

Comment:
[nit] I don't understand the comment. Why does this get created from the subAlgo parameter? It would be better to describe what it represents rather than what creates it.

Reply:
I was inspired by your suggestion and wrote
```
// SubAlgo defines the ORB_SLAM3 specific algorithms that we support
```